.github/workflows/e2e-test.yaml

```diff
@@ -42,6 +42,18 @@ jobs:
       - name: Create test output directory
         run: mkdir test/e2etest/artifacts
 
+      - name: Add private SSH key for Bitbucket tests
+        uses: shimataro/ssh-key-action@v2
+        with:
+          key: ${{ secrets.BITBUCKET_E2E_TEST_PRIV_SSH_KEY }}
+          name: bitbucket-e2e # optional
+          known_hosts: dummy-entry
+          if_key_exists: fail # replace / ignore / fail; optional (defaults to fail)
+
+      - name: Add public SSH key for Bitbucket tests
+        run: |
+          echo ${{ secrets.BITBUCKET_E2E_TEST_PUB_SSH_KEY }} >> /home/runner/.ssh/bitbucket-e2e.pub
```

Comment:
Technically we should use `>` to create/overwrite the file for public key. For this case `>>` (append) works as well because this is in a Github action and we know no file `/home/runner/.ssh/bitbucket-e2e.pub` existed. 

However, I say as an optional change better to use the right syntax and replace `>>` with `>`